macros/generate_source.sql

```diff
@@ -1,9 +1,9 @@
-{% macro get_tables_in_schema(schema_name, database_name=target.database) %}
-
+{% macro get_tables_in_schema(schema_name, table_pattern, database_name=target.database) %}
```

Comment:
If anyone is calling this directly with existing code, adding a new param that doesn't have a default would break it:

```suggestion
{% macro get_tables_in_schema(schema_name, table_pattern='%', database_name=target.database) %}
```

Also worth calling out that by putting this in second position instead of adding it to the end will also be a breaking change for anyone using positional arguments instead of naming them. I think it's the right thing to do because it's then consistent with the dbt_utils macro's default order, but it does mean this will be a minor release instead of a patch release.

Reply:
Thinking about it more, I think it'd be worth adding support for `exclude` while you're here, so that people can use everything that's available to dbt utils.